CMakeLists.txt

```
@@ -38,6 +38,7 @@ option(CONTOUR_FRONTEND_GUI "Enables GUI frontend." ON)
 option(CONTOUR_COVERAGE "Builds with codecov [default: OFF]" OFF)
 option(CONTOUR_SANITIZE "Builds with Address sanitizer enabled [default: OFF]" OFF)
 option(CONTOUR_STACKTRACE_ADDR2LINE "Uses addr2line to pretty-print SEGV stacktrace." ${ADDR2LINE_DEFAULT})
+option(CONTOUR_BUILD_WITH_EMBEDDED_HARFBUZZ "Builds with embedded FreeType and HarfBuzz [default: OFF]" OFF)
```

```suggestion
option(CONTOUR_BUILD_WITH_EMBEDDED_FT_HB "Builds with embedded FreeType and HarfBuzz [default: OFF]" OFF)
```

I think it is best to have **both** libraries somehow referenced with their well known shortcuts (FT, HB). Otherwise it feels like magic that it's indicating not as much as it's inducing. :)

---

Done.